pom.xml

```diff
@@ -246,6 +246,12 @@
                   <ignoreMissingAnnotations>false</ignoreMissingAnnotations>
                 </missing-classes>
                 <filter>
+X                  <classes>
```

Comment:
`X` marks the spot? :smile:

Reply:
thats strange it worked, but now i know where my X went)